docs/control-structures.rst

```diff
@@ -28,6 +28,9 @@ something like::
         }
     }
 
+...note::
+    A function cannot accept a multi-dimensional array as an input parameter. This functionality is possible if you enable the experimental ``ABIEncoderV2`` feature by adding ``pragma experimental ABIEncoderV2;`` to your smart contract.
```

Comment:
I think "experimental ABIEncoderV2 feature" should be written as something like "the new experimental ABI encoder implementation".

Also it should link to the ABI section and it probably should have a more generic intro (it is not only for multi dimensional arrays, but a bunch of things).

Reply:
Done